tests/zenko_tests/node_tests/utils/getWebIdentityToken.js

```diff
@@ -0,0 +1,85 @@
+const querystring = require("querystring");
+const https = require("https");
+const assert = require('assert');
+
+const USER_1_PASSWORD = process.env.KEYCLOAK_TEST_PASSWORD || '123';
+const HOST_1_URL = process.env.KEYCLOAK_TEST_HOST || 'https://keycloak.zenko.local';
+const HOST_1_PORT = parseInt(process.env.KEYCLOAK_TEST_PORT, 10) || 443;
+const REALM_NAME = process.env.KEYCLOAK_TEST_REALM_NAME || 'zenko';
+const KEYCLOAK_PATH = `/auth/realms/${REALM_NAME}/protocol/openid-connect/token`;
+const CLIENT_ID = process.env.KEYCLOAK_TEST_CLIENT_ID || 'zenko-ui';
+const GRANT_TYPE = process.env.KEYCLOAK_TEST_GRANT_TYPE || 'password';
+
+
+function getTokenForIdentity(identity, callback) {
+    getWebIdentityToken(identity, USER_1_PASSWORD, HOST_1_URL,
+        HOST_1_PORT, KEYCLOAK_PATH, CLIENT_ID, GRANT_TYPE, (err, token) => {
+            assert(err === null);
+            callback(err, token);
+        });
+}
+
+
+/**
+ * HTTP client to request JWT token given the username and password.
+ *
+ * @param {string} username - username of user requesting token
+ * @param {string} password - password of user requesting token
+ * @param {string} host - host URL of keycloak service
+ * @param {number} port - port of keycloak service
+ * @param {string} path - path of keycloak service authentication API
+ * @param {string} client_id - id of the client of the user
+ * @param {string} grant_type - grant of the user
+ * @param {function} callback - callback function called with error or result
+ * @returns {undefined} undefined
+ */
+function getWebIdentityToken(username, password, host, port, path,
```

Comment:
This function is already in VaultClient that is in the dependencies, how about using the VaultClient function instead? If we want to change the function, it is easier to change it in one place only

Reply:
If the current VaultClient version does not embedded the function, we can bump it in package.json

Reply:
Have discussed with William privately as well, `getWebIdentityToken` is not in VaultClient